learn/xcm/overview.md

```diff
@@ -0,0 +1,138 @@
+
+**Developers must understand that wrongfully setting the XCM message can result in the loss of funds**. Consequently, it is essential to test XCM features on a TestNet before moving to a production environment.
+
+## Chain Registration
+
+Before two chains can start communicating, a messaging channel must be opened. Channels are unidirectional, meaning that a channel from chain A to chain B will only pass messages from A to B. Consequently, asset transfers will be possible only from chains A to B. Therefore, two channels must be opened to send messages (or transfer assets) back and forth.
+
+A channel for XCMs between the relay chain and parachain is automatically opened when a connection is established. However, when parachain A wants to open a communication channel with parachain B, parachain A must send an open channel extrinsic in its network. This extrinsic is an XCM as well! The recipient of this XCM is the relay chain, and the extrinsic includes information such as:
+
+- The destination where the message will be executed (relay chain in this case)
+- The account that will pay the fees (paid in the relay chain token)
+- Fees that the transaction can consume when executed
+- Encoded call data, obtained by mimicking the extrinsic on the relay chin. This includes the following encoded information:
+    * Method to be called in the relay chain (open channel)
+    * Parachain ID of the target chain (parachain B in this example)
+    * Maximum number of messages in the destination queue
+    * Maximum size of the messages to be sent
+
+The transaction fees are paid in the cross-chain (xc) representation of the relay chain asset (_xcRelayChainAsset_). For example, for Kusama/Moonriver, the transaction fees would be paid in _xcKSM_. Therefore, the account paying for the fees must have enough _xcRelayChainAsset_. This might be tackled in Moonbeam/Moonriver by having fees from incoming XCM messages, which are paid in the origin chain asset, sent to the treasury, and using the treasury account to pay for the channel registration extrinsic.
+
+Even though parachain A has expressed its intentions of opening and XCM channel with parachain B, the latter has not signaled to the relay chain its intentions to receive messages from parachain A. Therefore, to have an established channel, parachain B must also send an extrinsic (which is also an XCM) to the relay chain. The accepting channel extrinsic is similar to the previous one. However, the encoded call data only includes the new method (accept channel) and the parachain ID of the sender (parachain A in this example). Once both parachains have agreed, the channel is opened within the next epoch change.
+
+![XCM Channel Registration Overview](/images/learn/xcm/overview-3.png)
```

Comment:
I'm not really a fan of the Moonbeam logo on top of the person to represent the Sovereign account. To me it just looks like a regular user in the logo. Can we change it to something that more represents a group / sovereign entity such as a democracy / council / state related logo?

Reply:
The thing is that it is an account owned by the Moonbeam parachain